### ISSUE TITLE
Are there pre-trained topic models?

### ISSUE THREAD

**Issue by nishkalavallabhi** [chaitaliSaini/gensim-data#30]
Like pre-trained word embeddings, are there any pre-trained topic models available (now or in future) in gensim?

**Comment by menshikh-iv** [comment]
@nishkalavallabhi yes, this is in our plan, ideally, we want to have several variants for **each** model available in Gensim
